content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,382 @@
+     :guilabel:`Goods` as **tax scope**.
+
+#. `TD19` - Buying **goods** from a **foreign** vendor, but the goods are already in Italy in a VAT
+   deposit:
 
-You can check the current status of your customer invoice under the **Electronic invoicing** field.
+   - From EU: the *buyer* integrates the invoice received for VAT;
+   - Non-EU: the *buyer* sends an invoice to *themselves* (i.e., **self-billing**).
 
-.. image:: italy/processing.png
+     Odoo exports a move as a `TD19` if the following conditions are met:
+
+  - Is a vendor bill;
+  - At least one tax on the invoice lines targets the tax grid `VJ3`;
+  - All invoice lines either have :guilabel:`Consumable` products, or a tax with
+    :guilabel:`Goods` as tax scope.
+
+.. _italy/internal-reverse:
+
+Internal reverse charge
+-----------------------
+
+.. important::
+   Odoo currently does not support **internal reverse** charge processes.
+
+In some cases, vendor bills have the VAT duty **reverse charged** on the *buyer* for **national
+purchases** too (e.g., cleaning company invoices, construction sub-contractor invoices, scrap
+selling, tablets and mobile phones, etc.). For these operations, the *seller* or *provider* issues
+an electronic invoice with :guilabel:`Tipo Documento` code `TD01`, `TD02` for :ref:`down payments
+<italy/down-payments>`, or `TD04` for credit notes, with the **tax-exemption code** ranging from
+`N6.1` to `N6.9`.
+
+The *buyer* is required to issue themselves an invoice (i.e., **self-billing**) for the VAT due with
+:guilabel:`Tipo Documento` code `TD16`, which is sent through the :abbr:`SdI (Systema di
+Interscambio)` with the buyer *themselves* as **recipient** (using their own :guilabel:`Codice
+Destinatario`). That way, they can deduct their VAT. The **self-billed** invoice must be issued and
+recorded in the VAT sales register within **15 days** from receipt.
+
+Customer invoices
+=================
+
+The :guilabel:`Fattura PA` feature is enabled by default, and can be found under
+:menuselection:`Accounting --> Configuration --> Journals --> Customer Invoices (journal) -->
+Advanced Settings (tab) --> Electronic Data Interchange`. Once :doc:`created and confirmed
+<../../getting_started/process_overview/customer_invoice>`, the e-Invoice is generated and sent. You
+can check the current status of an invoice by the :guilabel:`Electronic invoicing` field *(cf.
+screenshot below)*. The XML file can be found in the **chatter** of the invoice.
+
+.. warning::
+   e-Invoices are only sent to the government if the :abbr:`EDI (Electronic Document Invoicing)` is
+   set to :guilabel:`Official` mode.
+
+.. image:: italy/italy-processing.png
    :align: center
    :alt: Electronic invoicing status (waiting for confirmation)
 
+.. _italy/down-payments:
+
+Down payments
+-------------
+
+**Down payments** invoices are imported/exported with a different :guilabel:`Typo Documento` code
```

Comment:
There's a typo, it is `Tipo Documento` not `Typo Documento`. 
Which is kinda recursive. 🤣